### PR TITLE
udpate zh-trans content/zh/docs/tasks/administer-cluster/topology-man…

### PR DESCRIPTION
**Pull request by tanjunchen**
ref:https://github.com/k8smeetup/website-tasks/issues/3700
/cc @tengqm

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/topology-manager.md [hunk 1]
```diff
@@ -180,6 +180,13 @@ Topology Manager will reject this pod from the node. This will result in a pod i
 如果亲和性不是首选，则拓扑管理器将从节点中拒绝此 Pod 。
 这将导致 Pod 处于 `Terminated` 状态，且 Pod 无法被节点接纳。
 
+<!--
+Once the pod is in a `Terminated` state, the Kubernetes scheduler will **not** attempt to reschedule the pod. It is recommended to use a ReplicaSet or Deployment to trigger a redeploy of the pod.
+An external control loop could be also implemented to trigger a redeployment of pods that have the `Topology Affinity` error.
+-->
+一旦 Pod 处于 `Terminated` 状态，Kubernetes 调度器将不会尝试重新调度改 Pod。建议使用 ReplicaSet 或者 Deployment 来重新部署 Pod。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/topology-manager.md]:
```suggestion
一旦 Pod 处于 `Terminated` 状态，Kubernetes 调度器将不会尝试重新调度该 Pod。建议使用 ReplicaSet 或者 Deployment 来重新部署 Pod。
```

File: content/zh/docs/tasks/administer-cluster/topology-manager.md [hunk 2]
```diff
@@ -180,6 +180,13 @@ Topology Manager will reject this pod from the node. This will result in a pod i
...
+还可以通过实现外部控制循环，以触发具有 `Topology Affinity` 错误的 Pod 的重新部署。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/topology-manager.md]:
```suggestion
还可以通过实现外部控制环，以启动对具有 `Topology Affinity` 错误的 Pod 的重新部署。
```

File: content/zh/docs/tasks/administer-cluster/topology-manager.md [hunk 3]
```diff
@@ -205,6 +212,12 @@ If, however, this is not possible then the Topology Manager will reject the pod
 如果不可能，则拓扑管理器将拒绝 Pod 运行于该节点。
 这将导致 Pod 处于 `Terminated` 状态，且 Pod 无法被节点接受。
 
+<!--
+Once the pod is in a `Terminated` state, the Kubernetes scheduler will **not** attempt to reschedule the pod. It is recommended to use a Deployment with replicas to trigger a redeploy of the Pod.
+An external control loop could be also implemented to trigger a redeployment of pods that have the `Topology Affinity` error.
+-->
+一旦 Pod 处于 `Terminated` 状态，Kubernetes 调度器将不会尝试重新调度改 Pod。建议使用 ReplicaSet 或者 Deployment 来重新部署 Pod。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/topology-manager.md]:
```suggestion
一旦 Pod 处于 `Terminated` 状态，Kubernetes 调度器将不会尝试重新调度该 Pod。建议使用 ReplicaSet 或者 Deployment 来重新部署 Pod。
```

File: content/zh/docs/tasks/administer-cluster/topology-manager.md [hunk 4]
```diff
@@ -205,6 +212,12 @@ If, however, this is not possible then the Topology Manager will reject the pod
...
+还可以通过实现外部控制循环，以触发具有 `Topology Affinity` 错误的 Pod 的重新部署。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/topology-manager.md]:
```suggestion
还可以通过实现外部控制环，以触发具有 `Topology Affinity` 错误的 Pod 的重新部署。
```

File: content/zh/docs/tasks/administer-cluster/topology-manager.md [hunk 5]
```diff
@@ -274,19 +287,62 @@ This pod runs in the `Guaranteed` QoS class because `requests` are equal to `lim
...
 
 <!--
-Topology Manager will use this information to store the best Topology for this container. In the case of this Pod, CPU and Device Manager will use this stored information at the resource allocation stage.
+The Topology Manager would consider both of the above pods. The Topology Manager would consult the Hint Providers, which are CPU and Device Manager to get topology hints for the pods. 
+In the case of the `Guaranteed` pod the `static` CPU Manager policy would return hints relating to the CPU request and the Device Manager would send back hints for the requested device.
 -->
-拓扑管理器将使用此信息来存储此容器的最佳拓扑。
-对于此 Pod， CPU 和设备管理器将在资源分配阶段使用这里存储的信息。
+拓扑管理器将考虑以上两个 Pod。拓扑管理器将咨询 CPU 和设备管理器，以获取 Pod 的拓扑提示。
+对于 `Guaranteed` Pod，`静态` CPU 管理器策略将返回与 CPU 请求有关的提示，而设备管理器将返回有关所请求设备的提示。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/topology-manager.md]:
```suggestion
对于 `Guaranteed` Pod，`static` CPU 管理器策略将返回与 CPU 请求有关的提示，而设备管理器将返回有关所请求设备的提示。
```

**Comment by tanjunchen** [comment]
/cc @tengqm
done

**Pull request merged**